agent/engine/execcmd/manager_init_task_linux.go

```diff
@@ -0,0 +1,151 @@
+// +build linux
```

Comment:
different methods implementation (init task/start agent) could live in the same file? `manager_linux.go`?

Reply:
They could, it's a matter of preference. I prefer not to have to much unrelated code in the same file and keep files short and readable.

Reply:
its ok to keep it this way, just that I haven't seen examples like this in our code base. not sure if there is a best practice